tests/src/3d/testqgs3drendering.cpp

```diff
@@ -1403,9 +1403,9 @@ void TestQgs3DRendering::testDepthBuffer()
 
   scene->cameraController()->depthBufferCaptured( depthImage );
 
-  QGSCOMPARENEARVECTOR3D( testCam->zoomPoint(), QVector3D( -32.4, 222.8, 184.1 ), 1.0 );
-  QGSCOMPARENEARVECTOR3D( testCam->cameraPose()->centerPoint(), QVector3D( -32.4, 222.8, 184.1 ), 1.0 );
-  QGSCOMPARENEAR( testCam->cameraPose()->distanceFromCenterPoint(), 126.4, 0.1 );
+  QGSCOMPARENEARVECTOR3D( testCam->zoomPoint(), QVector3D( -40.3, 276.8, 228.8 ), 1.0 );
+  QGSCOMPARENEARVECTOR3D( testCam->cameraPose()->centerPoint(), QVector3D( -40.3, 276.8, 228.8 ), 1.0 );
+  QGSCOMPARENEAR( testCam->cameraPose()->distanceFromCenterPoint(), 114.6, 0.1 );
```

Comment:
why is there change in test results here, and in the test control images?

Reply:
Because the test was broken. The far plane taken into account was wrong.